assets/LICENSE

```diff
@@ -0,0 +1,686 @@
+                    Radicle Linking Exception
```

Comment:
In the smart contracts we're using the SPDX license headers. I wonder if [SPDX-License-Identifier: (GPL-3.0-only WITH GPL-3.0-linking-exception)](https://spdx.org/licenses/GPL-3.0-linking-exception.html) will be compatible with this license?

Reply:
I'm going to ping Ange, since she's the one advising me on the legalese

Reply:
Thanks! If it's good, we could use it in all the repos, it's much lighter than our custom header

Reply:
Andrew mentioned in [his post](https://radicle.community/t/radicle-licensing-model/282), but with regards to the Radicle Linking Exception: "Unfortunately, there is no current SPDX identifier for this exception, but we are looking to apply for one from SPDX." So while it's heavier, let's use our custom header at this time - though we will hopefully be able to go lighter in the future.

Reply:
@CodeSandwich Sorry, I may have misunderstood your comment here! Yes, while we can't use an SPDX identifier at this time, we don't need to use the WHOLE wrapper about the Radicle Linking Exception in each source file :)